core/project.clj

```diff
@@ -4,4 +4,5 @@
   :scm {:dir ".."}
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.7.0"]])
+  :dependencies [[org.clojure/clojure "1.7.0"]]
+  :plugins [[lein-cljfmt "0.9.0"]])
```

Comment:
Remove this plugin, as it's not relevant to the feature.

Reply:
😱